samples/bookinfo/platform/kube/rbac/upgradeToClusterRbacConfig.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
You should keep the script under tool folder, or a better place..
A nicer option is having this in istioctl (or auto-convert in pilot), though the cost is obviously higher.

Reply:
Moved to under tool folder. I'll look into istioctl to also provide this.